packages/main/src/Card.js

```diff
@@ -133,6 +167,42 @@ class Card extends WebComponent {
 
 		super.define(...params);
 	}
+
+	headerPress(event) {
+		const click = event.type === "click";
```

Comment:
I am not really sure how stable is this.

Reply:
Yes, this approach is arguable, we can discuss it briefly today.